src/libm/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+# Helper executable: generates parts of the sleef header file
+add_executable(${TARGET_MKRENAME} mkrename.c)
+target_compile_options(${TARGET_MKRENAME}
+  PRIVATE ${FLAGS_BASIC_C} ${FLAGS_WALL})
+
+# Helper executable: dispatcher
+add_executable(${TARGET_MKDISP} mkdisp.c)
+target_compile_options(${TARGET_MKDISP}
+  PRIVATE ${FLAGS_BASIC_C} ${FLAGS_WALL})
+
+# TODO: set the parameters for mkrename run
+set(MKRENAME_PARAMS "")
```

Comment:
I believe this is needed only for the SIMD implementation - can you update your comment?

```
set the parameters for mkrename runs for the SIMD implementation
```